packages/ember-application/lib/system/application.js

```diff
@@ -1003,6 +1004,7 @@ Application.reopenClass({
 
     registry.injection('view', 'renderer', 'renderer:-dom');
     registry.register('view:select', SelectView);
+    registry.register('view:outlet', OutletView);
```

Comment:
I am unsure if the others are public api either. But regardless this offers a mild buffer from resolved alternatives.